acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -115,11 +120,6 @@ func TestAPIGateway_Basic(t *testing.T) {
 				k8s.RunKubectlAndGetOutputE(t, ctx.KubectlOptions(t), "delete", "-f", "../fixtures/cases/api-gateways/tcproute/route.yaml")
 			})
 
-			// We use the static-client pod so that we can make calls to the api gateway
-			// via kubectl exec without needing a route into the cluster from the test machine.
-			logger.Log(t, "creating static-client pod")
-			k8s.DeployKustomize(t, ctx.KubectlOptions(t), cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory, "../fixtures/bases/static-client")
-
```

Comment:
~ Why the relocation above for this block?

Reply:
I was shuffling things around to add retries & checks to the deployments but it didn't work out.

Reply:
Oh, like we don't need to move this after all?

Reply:
Not really. It'll give it a few ms to deploy and get ready but it shouldn't make a difference.